src/index.ts

```diff
@@ -21,9 +21,9 @@ const readFile = fs.readFile
       );
     };
 
-const GOOGLE_TOKEN_URL = 'https://www.googleapis.com/oauth2/v4/token';
+const GOOGLE_TOKEN_URL = 'https://oauth2.googleapis.com/token';
 const GOOGLE_REVOKE_TOKEN_URL =
-  'https://accounts.google.com/o/oauth2/revoke?token=';
+  'https://oauth2.googleapis.com/token/revoke?token=';
```

Comment:
where did you get this revoke url from?

Reply:
In the documentation it said that the revoke old url (accounts.google.com) was `extremely deprecated`, so I browsed through the proto from the yaqs issue and saw revoke was in there.

Reply:
Got it. As long as it works, it should be ok.